syntax/element_content.ml

```diff
@@ -228,6 +228,8 @@ let script ~lang ~loc ~name children =
     [Nolabel, child]
   | _ -> Common.error loc "%s can have at most one child" name
 
+let textarea = script
```

Comment:
I would prefer you pulled the function as a `at_most_one_children` function, and alias that both time. :)

Reply:
Oh, right, that's way clearer.